apps/api-gateway/src/oid4vc-verification/dtos/verify-authorization-response.dto.ts

```
@@ -0,0 +1,36 @@
+  })
+  @IsString()
+  @IsNotEmpty()
+  verificationSessionId!: string;
+
+  @ApiProperty({
+    description:
+      'authorization response received from the wallet after user approves or denies the verification request',
+    example: {
+      // eslint-disable-next-line camelcase
+      id_token: {
+        Passport: [
```

ESLint

[error] There should be no linebreak after '['.

(array-bracket-newline)